lib/mongo/protocol/msg.rb

```diff
@@ -182,6 +182,36 @@ def documents
         [@main_document]
       end
 
+      def maybe_encrypt(client)
```

Comment:
Can this and maybe_decrypt methods have docstrings added please?

Reply:
I'm going to do this right now in a follow-up PR.